src/store.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import restaurantReducer from './pages/restaurant/reducer/restaurant.reducer';
```

Comment:
keep a reducers folder at root level and inside it create an index.js file....inisde that combine the reducers and import it here